corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py

```diff
@@ -99,6 +106,60 @@ def migrate_app(self, app_id):
         app.save()
         return True
 
+    def fix_user_properties(self, app_id):
+        app = Application.get(app_id)
+        if not should_fix_user_props(app):
+            return False
+        logger.info('fixing app %s', app_id)
+        modules = [m for m in app.modules if m.module_type == 'basic']
+        updated = False
+        for module in modules:
+            forms = [f for f in module.forms if f.doc_type == 'Form']
```

Comment:
i don't have a lot of context here, but do you need to consider other subclasses of FormBase, or are you explicitly filtering those out